backend/src/controllers/userController.ts

```diff
@@ -103,15 +103,40 @@ export const updateUser = async (req: Request, res: Response) => {
   }
 };
 export const getAllUsers = async (req: Request, res: Response) => {
-  const { page, limit } = req.query;
+  const { page, limit, ownerAddress, query } = req.query;
```

Comment:
Use req.body.address 
Look at this PR to get an idea
https://github.com/globe-and-citizen/cnc-portal/pull/168